app/controllers/user_course_subjects_controller.rb

```diff
@@ -0,0 +1,72 @@
+class UserCourseSubjectsController < ApplicationController
+  before_action :logged_in_user
+  before_action :load_user_course_subject, :user_course_subject_params
+  before_action :correct_user
+  before_action :course_cancel?
+  before_action :subject_inprogress?, :first_subject_start?
+
+  def update
+    if @user_course_subject.start? && params[:status]
+      start_time_params = {start_time: Time.current}
+      @user_course_subject_params.merge start_time_params
+    end
+
+    if @user_course_subject.update @user_course_subject_params
```

Comment:
```suggestion
    elsif @user_course_subject.update @user_course_subject_params
```
Thử như này xem được không em

Reply:
Không anh ơi, cái if phía trên là nếu thoả mãn điều kiện bắt đầu 1 môn học đầu tiên thì cho thêm trường start_time vào để tính thời gian bắt đầu, sau đó cập nhật status và start_time. Còn không thì chỉ cập nhật status thôi ạ